src/main.cc

```diff
@@ -810,6 +819,12 @@ sig_child(int sig)
 static void
 serverConnectionsOpen(void)
 {
+    if (Store::Controller::StoreForegroundRebuilding()) {
+        if (IamPrimaryProcess() || IamWorkerProcess())
+            debugs(1, DBG_IMPORTANT, "Will wait for Store indexing completion before opening listening sockets");
```

Comment:
Let's use a shorter version and better accommodate the cases where this function is called many times while store is still rebuilding:
```suggestion
            debugs(1, DBG_IMPORTANT, "Waiting for Store indexing completion before opening listening sockets");
```